coremlpython/CoreMLPythonUtils.mm

```diff
@@ -4,6 +4,15 @@
 #include <pybind11/eval.h>
 #include <pybind11/numpy.h>
 
+#include <numpy/arrayobject.h>
```

Comment:
This should be moved into the `PY_MAJOR_VERSION=3`, since numpy is not needed in Python 2. I will push a commit that moves this in.